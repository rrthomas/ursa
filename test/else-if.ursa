for i in range(10) {
  if i == 0 { print("nought") }
  else if i == 1 { print("one") }
  else if i == 2 { print("two") }
  else { print("many") }
}
