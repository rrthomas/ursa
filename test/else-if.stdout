nought
one
two
many
many
many
many
many
many
many